### DISCUSSION TITLE
Frigate LXC script finishes with error - container seems to be created but nothing is accessible

### DISCUSSION THREAD

**Discussion by Menz01** [tteck/Proxmox#3825] [Help Needed]
Hello,
    i am running the Frigate LXC script and using the advanced options so i can resize some things. the script finished and said there was an error. the container still gets created and i see the IP in my router, however, when i try to access the page its not there. here is the error i am seeing..

![image](https://github.com/user-attachments/assets/0b30d0d2-4237-4395-9f5e-e4eda52ddfc0)

**Comment by tteck** [comment]
🛑 Failure to comply with the following guideline.
> - If the default Linux distribution is not adhered to, script support will be discontinued.
